Mods/vcmi/config/vcmi/english.json

```diff
@@ -50,13 +50,12 @@
 	"vcmi.adventureOptions.forceMovementInfo.help" : "{Always show move info in status bar}\n\n Replaces default status bar info with movement points data without need to hold ALT button.",
 	"vcmi.adventureOptions.showGrid.hover" : "Show grid",
 	"vcmi.adventureOptions.showGrid.help" : "{Show grid}\n\n Shows grid overlay, showing borders between adventure map tiles.",
-	"vcmi.adventureOptions.mapScrollSpeed4.hover": "4",
 	"vcmi.adventureOptions.mapScrollSpeed4.help": "Set map scrolling speed to very fast",
-	"vcmi.adventureOptions.mapScrollSpeed5.hover": "5",
 	"vcmi.adventureOptions.mapScrollSpeed5.help": "Set map scrolling speed to extremely fast",
+	"vcmi.adventureOptions.mapScrollSpeed5.help": "Set map scrolling speed to fast, very fast.",
```

Comment:
duplicated key

Reply:
FIxed, will commit later.
Not bothering with localization at the moment since I want to review these settings anyway - e.g. shift all texts and add "very slow" as 1st option instead of coming up with more synonyms for "fast"